web/src/composables/useLogs.ts

```
@@ -1808,6 +1811,89 @@
+              searchObj.data.queryResults.hasOwnProperty("hits")
+            ) {
+              const start_time: number =
+                initialQueryPayload.value?.query?.start_time || 0;
+              const end_time: number =
+                initialQueryPayload.value?.query?.end_time || 0;
+              queryReq.query.start_time = start_time;
+              queryReq.query.end_time = end_time;
+            }
+          }
+        }
+        delete queryReq.aggs;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)